cli.js

```diff
@@ -102,25 +104,20 @@ engine(
     extensions: extensions,
     configTransform: transform,
     output: false,
-    out: false,
```

Comment:
I think this is causing your error. The `out` option is about writing files I think, not about the report!

Reply:
thanks, yes that was the problem. Fixed